README.md

```diff
@@ -34,33 +34,17 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
-The `.env` file also stores Trello credentials which are required to run the app. You will need a [Trello account](https://trello.com/signup) and API key. You can find instructions on how to do this [here](https://trello.com/app-key). Replace the following environment variables in your `.env` file:
+The `.env` file also stores Mongo DB credentials which are required to run the app. It is recommended to use the Mongo DB Comos DB in Azure. You can find instructions on how to do this [here](https://learn.microsoft.com/en-us/azure/cosmos-db/mongodb/quickstart-dotnet?tabs=azure-cli&pivots=devcontainer-codespace). Replace the following environment variables in your `.env` file:
 
 ```bash
-TRELLO_API_KEY=trello-api-key
-TRELLO_SECRET=trello-secret
+MONGO_CONNECTION_STRING=[mongo-connection-string]
+MONGO_DATABASE_NAME=[db-name]
+TRELLO_API_KEY=[trello-api-key]
+TRELLO_SECRET=[trello-secret]
+TRELLO_BOARD_ID=[trello-board-id]
```

Comment:
Worth asking whether you still want to document these, they're required for completing the migration, but maybe that's something we expect to be completed on a timetable so that this is quickly going to be out of date? 🤔